pre-piksi-multi-launch-updates.md

```diff
@@ -0,0 +1,121 @@
+### SBP changes before shipping Piksi Multi
+
+There are a few changes we would like to make in the SBP protocol. These changes
+will be much more painful to make after we ship and customers are relying on
+the protocol to be largely static, so we'd like to come to a consensus on
+these and implement them before shipping Piksi Multi.
+
+These changes attempt to accomplish a few things:
+- Make messages more bandwidth efficient
+- Make user side logic of using messages more obvious
+- Remove state from the protocol, where possible
+
+### 1. Replace obs forwarding with general purpose forwarding message
+
+Current there is a convention in the protocol that if the sender_id of a
```

Comment:
Would be good to write down various use cases when messages are forwarded and used.
Would be good to know from what port data is forwarded from.
We should make it to be able to forward data in other protocols (NMEA0183, NMEA2000, RTCMv3,...).